src/main/java/org/broadinstitute/hellbender/engine/spark/AuthHolder.java

```diff
@@ -0,0 +1,141 @@
+package org.broadinstitute.hellbender.engine.spark;
```

Comment:
Should probably not live in `engine.spark`, since it's not spark-specific. Perhaps either `engine` or `utils.gcs`?